Content.Server/Nutrition/Components/SliceableFoodComponent.cs

```diff
@@ -65,8 +65,8 @@ async Task<bool> IInteractUsing.InteractUsing(InteractUsingEventArgs eventArgs)
             {
                 return false;
             }
-
-            if (!eventArgs.Using.TryGetComponent(out UtensilComponent? utensil) || !utensil.HasType(UtensilType.Knife))
+            
+            if (!eventArgs.Using.TryGetComponent(out UtensilComponent? utensil) || !utensil.Types.HasFlag(UtensilType.Knife))
```

Comment:
use `& UtensilType.Knife` instead of HasFlag

Reply:
(The reason for this is because HasFlag allocates in debug for reasons)